csi.proto

```diff
@@ -305,11 +293,7 @@ message DeleteVolumeRequest {
 message DeleteVolumeResponse {
   message Result {}
 
-  // One of the following fields MUST be specified.
-  oneof reply {
-    Result result = 1;
-    Error error = 2;
-  }
+  Result result = 1;
```

Comment:
When you have an empty message, go ahead and use `google.protobuf.Empty`.

Was there a reason for the extra encapsulation here?

Reply:
Will do.